### PR TITLE
Prepare repository for build with Jenkins pipelines

### PR DESCRIPTION
**Pull request by simonrankine**

### CODE REVIEW

File: Jenkinsfile
Comment (samdbmg) [inline Jenkinsfile]: Nitpick: Indentation (43-45)

File: Jenkinsfile
Comment (samdbmg) [inline Jenkinsfile]: Nitpick: Indentation

File: Jenkinsfile
Comment (samdbmg) [inline Jenkinsfile]: After a discussion with @j616 it might be better to use the variable in a script pattern seen elsewhere, since this is the only place this method shows up. More generally, https://github.com/bbc/rd-apmm-groovy-ci-library/commit/0d8a57c60e06274f4a3681a46d88bd049a1243bd contains some changes to restructure the parallel stages and build wheels and packages together. Up to you whether to pull those in.

File: Makefile
Comment (samdbmg) [inline Makefile]: Does the Python 3 package actually work? See also #9 where I tried to get it working, but it's blocked on a bunch of other stuff in nmoscommon working first, and python3-pybonjour to work and python3-avahi to actually exist. Might be worth removing from here and the wheel build if it doesn't work...

File: Makefile
Comment (samdbmg) [inline Makefile:10]: You're not going to like this, but because of the way py2dsc works (well, I assume it's py2dsc doing it...never worked it out!) you end up with a package called python-python-nodefacade (see https://jenkins.rd.bbc.co.uk/blue/organizations/jenkins/apmm-repos%2Fnmos-node/detail/PR-12/1/artifacts). I think either the name in setup.py needs to become "nodefacade" (and accept the potential breakage that ensues) or some more Makefile hacking is needed.

File: Jenkinsfile
Comment (samdbmg) [inline Jenkinsfile]: See other comment about broken Python 3 package

**Review by samdbmg — CHANGES_REQUESTED** [review]
Package names are a bit weird at the moment - mostly because a Python module name shouldn't start with "python-"

**Comment by andrewbonney** [comment]
Just a thought. This one is a clear candidate for an upstream trigger in the Jenkinsfile, coming from nmos-common.

File: Jenkinsfile
Comment (simonrankine) [reply Jenkinsfile]: Hmm, some terrible tabs/spaces shenanigans going on here...

**Review by samdbmg — APPROVED** [review]

**Comment by samdbmg** [comment]
Looks good, once CI passes

**Comment by samdbmg** [comment]
For monitoring upstream; have a look at https://github.com/bbc/rd-apmm-python-lib-mediatimestamp/commit/e92c8014762a31bb5d59caaf0e2091f85c8768d6

**Comment by simonrankine** [comment]
Note to self, check packages install okay before merging

**Pull request merged**
